default.nix

```diff
@@ -137,6 +137,9 @@ in rec {
           locations.${baseUrl} = {
             proxyPass = "http://127.0.0.1:" + toString internalPort;
             proxyWebsockets = true;
+            extraConfig = ''
+              access_log stdout;
```

Comment:
not sure how much this will spam your syslog, but i think it will just be info severity so that's okay

Reply:
let's just set it to off, since obelisk has its own access log

Reply:
```suggestion
              access_log off;
```